src/code/gfxprint.c

```diff
@@ -238,11 +236,11 @@ void GfxPrint_PrintChar(GfxPrint* this, u8 c) {
     u8 charParam = c;
 
     if (c == ' ') {
-        this->posX += 0x20;
-    } else if (c > 0x20 && c < 0x7F) {
+        this->posX += 32;
+    } else if (c > ' ' && c < 0x7F) {
         GfxPrint_PrintCharImpl(this, charParam);
     } else if (c >= 0xA0 && c < 0xE0) {
-        if (this->flag & GFXPRINT_FLAG1) {
+        if (this->flags & GFXP_FLAG_HIRAGANA) {
             if (c < 0xC0) {
                 charParam = c - 0x20;
             } else {
```

Comment:
Yeah I'm not familiar with the text format itself beyond the fact that this code somehow changes katakanas into hiraganas, but it could probably be documented clearly in the future

Reply:
I'll probably document it when I deal with the font.